services/slam/fake/slam.go

```diff
@@ -49,6 +49,10 @@ type SLAM struct {
 	logger    golog.Logger
 }
 
+func NewSLAM(name string, logger golog.Logger) *SLAM {
```

Comment:
@nicksanford looks like this would be possible, except for the -1 counter being set.  I don't think this is a big deal though, I'm inclined to leave it as is

Reply:
that's still fine, just change here to:
```suggestion
func NewSLAM(cfg config.Component, logger golog.Logger) (*SLAM, error) {
return &SLAM{Name: cfg.Name, logger: logger, dataCount:-1}, nil
```

And then line 36 to:
```
return NewSLAM(cfg, logger)
```

This is actually better for fake slam long term.